metals/src/main/resources/db/migration/V5__Show_bsp_errors.sql

```diff
@@ -0,0 +1,4 @@
+-- If should show bsp errors
+create table show_bsp_errors(
```

Comment:
We could add it to the notifications table, no?

Reply:
Somehow I wasn't aware we had such a generic mechanism for notifications.